src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -439,6 +439,11 @@ class BulletRigidObject : public BulletBase,
    */
   Magnum::Range3D getCollisionShapeAabb() const override;
 
+  /** @brief Object data: All components of a @ref RigidObjectType::OBJECT are
+   * wrapped into one @ref btRigidBody.
+   */
+  std::unique_ptr<btRigidBody> bObjectRigidBody_;
```

Comment:
What's the reason for making this public?

Reply:
BulletPhysicsManager needs to get this to create the constraints. Could have been a getter to a pointer or reference, but won't be exposed through the primary APIs anyway.